docs/samples/Microsoft.ML.Samples/Dynamic/MinMaxNormalizer.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+        // the alignment of the usings with the methods is intentional so they can display on the same level in the docs site.
+        using Microsoft.ML.Runtime.Data;
+        using Microsoft.ML.Runtime.Api;
+        using Microsoft.ML.Data;
+        using System;
+        using System.Collections.Generic;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public partial class TransformSamples
+    {
+        public static void MinMaxNormalizer()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext(seed: 1, conc: 1);
+
+            // Get a small dataset as an IEnumerable and convert it to an IDataView.
+            IEnumerable<SamplesUtils.DatasetUtils.SampleInfertData> data = SamplesUtils.DatasetUtils.GetInfertData();
+            var trainData = ml.CreateStreamingDataView(data);
+
+            // Preview of the data.
+            // Age  Case  Education  Induced     Parity  PooledStratum  RowNum  ...
+            // 26   1       0-5yrs      1         6         3             1  ...
+            // 42   1       0-5yrs      1         1         1             2  ...
+            // 39   1       0-5yrs      2         6         4             3  ...
+            // 34   1       0-5yrs      2         4         2             4  ...
+            // 35   1       6-11yrs     1         3         32            5  ...
+
+            // A pipeline for normalizing the Induced column.
+            var pipeline = ml.Transforms.Normalizer("Induced");
+            // The transformed (normalized according to Normalizer.NormalizerMode.MinMax) data.
+            var transformedData = pipeline.Fit(trainData).Transform(trainData);
+            // Getting the data of the newly created column, so we can preview it.
+            var normalizedColumn = transformedData.GetColumn<float>(ml, "Induced");
+
+            // A small printing utility.
+            Action<string, IEnumerable<float>> printHelper = (colName, column) =>
+            {
+                Console.WriteLine($"{colName} column obtained post-transformation.");
+                foreach (var row in column)
+                    Console.WriteLine($"{row} ");
+            };
+
+            printHelper("Induced", normalizedColumn);
+
+            // Preview of the data.
+            // Induced
```

Comment:
>Induced [](start = 15, length = 7)

nit: this doesn't match $"{colName} column obtained post-transformation."

Reply:
The col name is "Induced", see line 39. 

---
In reply to: [227974775](https://github.com/dotnet/machinelearning/pull/1321#discussion_r227974775) [](ancestors = 227974775)

Reply:
I meant if these comment are supposed to be the output from printHelper. If they are, then the headers don't match:
            // Preview of the data.
            // Induced

vs 

Console.WriteLine($"{colName} column obtained post-transformation.");


---
In reply to: [227982369](https://github.com/dotnet/machinelearning/pull/1321#discussion_r227982369) [](ancestors = 227982369,227974775)